src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/SlhDsa/SlhDsaImplementationTests.cs

```diff
@@ -38,6 +48,307 @@ private static void AssertSlhDsaIsOnlyPublicAncestor(Func<SlhDsa> createKey)
             Assert.Equal(typeof(SlhDsa), keyType);
         }
 
+        #region Roundtrip by exporting then importing
```

Comment:
Genuine curiosity: What value do you feel \#region provides?

In my experience, they get out of date very early, and then are worse than useless (because they're wrong).

I feel like I left on the "region folding" feature for like a week back when I first started large team development.  But since it was often wrong I turned it off... and have probably not bothered with \#region since.

My inclination is to say "\#region is useless, please don't use it".  But I'll start with "why? :smile:"

Reply:
Mostly for logical organization similar to splitting these tests out into a partial class but that seemed like overkill. It was also useful to just collapse this region after I finished writing all the roundtrip tests - I expect new tests in this file to be added at the end so it saves some scrolling (and figuring out where exactly to put them). I tried to put the regions only around regions that I think are not going to change or be added to and can just be ignored in the future when modifying the file. VSCode also puts the regions on the expanded scroll bar map view which is somewhat useful.

That being said, System.Collections uses regions a lot and I found that very annoying too because it has tiny regions and writing a new feature generally meant expanding them all anyway.

Reply:
I'm still in favor of "never add \#regions", but I'll hold my nose, I guess.